did-tezos/Cargo.toml

```diff
@@ -8,10 +8,17 @@ edition = "2018"
 secp256k1 = ["libsecp256k1", "ssi/secp256k1", "ssi/rand"]
 
 [dependencies]
-ssi = { path = "../", default-features = false }
-chrono = { version = "0.4", features = ["serde"] }
+ssi = { path = "../" }
+# This file needs to be edited if the user want to use ed25519-dalek instead of ring.
+# The UX should improve in the future: https://github.com/rust-lang/cargo/issues/3494
+# ssi = { path = "../", default-features = false, features = ["ed25519-dalek", "rand", "sha2"] }
```

Comment:
Doesn't leaving `ssi` with `default-features = false` give the flexibility of being able to use either `ring` or `ed25519-dalek`?

Reply:
So you mean having `did-tezos` import `ring` or `ed25519-dalek` directly and `ssi` will have access to them indirectly? I can imagine having versioning issues later on, do you think it's still worth it?

Reply:
`did-tezos` doesn't depend on `ring` or `ed25519-dalek` directly. But it relies on one of those being enabled in `ssi`. `ssi` has `ring` enabled by default, but that can be disabled and `ed25519-dalek` used instead. What versioning issues would you see happening?

Reply:
Maybe I don't understand how to implement that logic -- what I understand is that we want features in did-tezos to enable/disable features in ssi. Which at the moment is not possible, from what I found including that linked issue?

Reply:
Tracking this issue in https://github.com/spruceid/ssi/issues/113

Reply:
I'm an idiot, it's right there `"ssi/rand"`, will fix that shortly.

Reply:
Done in bda4161